core-standards.php

```diff
@@ -39,3 +39,14 @@ function classloader($class) {
 add_action('widgets_init', __NAMESPACE__ . '\Widgets\UserLoginFormWidget::init');
 add_action('init', __NAMESPACE__ . '\Plugin::init', 20);
 add_action('admin_init', __NAMESPACE__ . '\Admin::init');
+
+if (defined('WP_CLI') && WP_CLI) {
+  // Subcommands that extend an existing namespace wrongly replace the whole
+  // namespace, so each subcommand needs to be registered separately.
+  // @see https://make.wordpress.org/cli/handbook/commands-cookbook/#required-registration-arguments
+  // @see https://github.com/wp-cli/wp-cli/issues/1767
+  // @see https://github.com/wp-cli/wp-cli/issues/1795
+  // @see https://github.com/wp-cli/wp-cli/pull/1877
```

Comment:
Sorry, I don't really get the mentioned issues here. The issues are telling, that the subcommands are not listed when running `$ wp help`, which was fixed in v0.19.0. So why did you mention them here?

Also this paragraph can be found in the cookbook:
>All commands can be registered to their own top-level namespace (e.g. wp foo), or as subcommands to an existing namespace (e.g. wp core foo). For the latter, simply include the existing namespace as a part of the command definition.

Or is this working for parent commands but not already existing subcommands?

Reply:
This comment was in relation to extending a namespace like this:
```php
  \WP_CLI::add_command('user', __NAMESPACE__ . '\Commands\User');
```
But you're right, the documentation has been updated in the meantime and suggests to register subcommands as already done in this PR, so the comments can be removed.